az-digital/az_quickstart
# Closes #1917 Adding config schema for az_migration module.

## Description
Adding a schema file for az_migration module.

## Related issues
Closes #1917 

## How to test
[Enable config inspector](https://db2624a2-98f1-4795-a44e-58012aa2d3f5.probo.build/user?destination=admin/reports/config-inspector)
Search for az_migration
Check that config is correct.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
